recipe/meta.yaml

```diff
@@ -1,29 +1,20 @@
-{% set version = "4.1.4" %}
+{% set version = "4.1.5" %}
 
 package:
   name: zeromq
   version: {{ version }}
 
 source:
   fn: zeromq-{{ version }}.tar.gz
-  url: http://download.zeromq.org/zeromq-{{ version }}.tar.gz
-  sha1: b632a4b6f8a14390dc17824e37ff7b10831ce2b4
-  patches:
-    # Patch to avoid failing tests on Mac. See issues for details.
-    # https://github.com/zeromq/libzmq/issues/1509
-    # https://github.com/zeromq/libzmq/issues/1878
-    # https://github.com/zeromq/libzmq/issues/1464
-    - Makefile.am.mac.patch    # [osx]
-    # Patch to avoid a failing test on Linux. See issue for details.
-    # https://github.com/zeromq/libzmq/issues/1462
-    - Makefile.am.linux.patch  # [linux]
+  url: https://github.com/zeromq/zeromq4-1/archive/v{{ version }}.tar.gz
```

Comment:
We can get rid of this `4-1` with a little bit of jinja. Feel free to pull that from the other PR.